lib/wallaroo_labs/connector_protocol/connector_protocol.pony

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Once sources on multiple workers is merged to master, are we going to update the `ConnectorSource` to use this?

Reply:
LOL I thought that I'd copied it from the ConnectorSource dev branch.  Nope, I didn't.  I haven't examined the reason deeply, but I didn't feel the need to spell out the FSM stages when working on the downstream end of the connector protocol for the Wallaroo source end, but *did* for the Wallaroo sink end.  I've created ticket #2814.